modules/importing-repositories/procedure-managing-the-imported-repository-list.adoc

```diff
@@ -0,0 +1,30 @@
+[id="managing-the-added-repositories]
+= Managing the added repositories
+
+.Prerequisites
+* You have xref:importing-multiple-github-repositories[imported GitHub repositories].
+
+
+.Procedure
+. Click *Bulk Import* in the left sidebar to display all the current repositories that are being tracked as Import jobs, along with their status.
+
+Added:: The repository is added to the {product-short} catalog after the import pull request is merged or if the repository already contained a `catalog-info.yaml` file during the bulk import. 
+This can be the case after the import PR has been merged, or if the repository already had a `catalog-info.yaml` when it was bulk-imported.
+Note that it may take a few minutes for the entities to be available in the catalog.
+
+Waiting for approval:: There is an open pull request adding a `catalog-info.yaml` file to the repository.
+You can:
+* Click the *pencil icon* on the right to see details about the pull request or edit the pull request content right from {product-short}.
+* Delete the Import job, this closes the import PR as well.
+* To transition the Import job to the _Added_ state, merge the import pull request from the Git repository.
+
+Empty:: {product-short} is unable to determine the import job status because the repository is imported from other sources but does not have a `catalog-info.yaml` file and lacks any import pull request adding it.
+This can be the case if the repository was imported from other sources but does not have any `catalog-info.yaml` file and does not have any import PR adding it.
```

Comment:
This information is added to the above sentence. Please remove this sentence.

Reply:
```suggestion
```